File: WEEK1/DAY5/Prog1/P1.java
import java.util.*;
public class P1 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int N;
        System.out.println("Print all factors of N");
        System.out.print("Enter N: ");
        N=sc.nextInt();
        for(int i=1;i<=(int)Math.sqrt(N);i++){
            if(N%i==0){
                System.out.println(i);
            }
        }
        for(int i=(int)Math.sqrt(N);i>=1;i--){
            if ((N%i==0) && (i!=(N/i))){
                System.out.println(N/i);
            }
        }
        sc.close();
    }
}

File: WEEK1/DAY5/Prog1/P1.py
import math
print("Print all factors of N")
N = int(input("Enter N: "))
for i in range(1, int(math.sqrt(N)) + 1):
    if N % i == 0:
        print(i)
for i in range(int(math.sqrt(N)), 0, -1):
    if N % i == 0 and i != N // i:
        print(N // i)
